src/Etg.Yams/Etg.Yams.nuspec

```diff
@@ -2,7 +2,7 @@
 <package >
   <metadata>
     <id>Etg.Yams</id>
-    <version>1.0.14</version>
+    <version>1.1.0-test</version>
```

Comment:
there's a lot of references to 1.1.0-test, is that something that you want merged?